js/adapt-stateful-session.js

```diff
@@ -118,6 +119,50 @@ define([
 			this.setupEventListeners();
 		},
 
+		/*
+		* SCORM 1.2 requires that the identifiers in cmi.interactions.n.student_response for choice and matching activities be a character from [0-9a-z].
+		* When numeric identifiers are used this function attempts to map identifiers 10 to 35 to [a-z]. Resolves issues/1376.
+		*/
+		_checkResponse: function(response, responseType) {
+			if (!response) return response;
+			if (responseType != 'choice' && responseType != 'matching') return response;
+
+			var zero = '0',
+				nine = '9';
+				a = 'a';
+				z = 'z';
+				base = a.charCodeAt(0);
+
+			response = response.split(/,|#/);
+
+			if (responseType == 'choice') {
+				response = response.map(checkIdentifier);
+			} else {
+				response = response.map(function(r) {
+					var identifiers = r.split('.');
+					return checkIdentifier(identifiers[0]) + '.' + checkIdentifier(identifiers[1]);
+				});
+			}
+
+			function checkIdentifier(r) {
+				var i;
+
+				// if [0-9] then ok
+				if (r.length == 1 && r >= zero && r <= nine) return r;
+
+				// if [a-z] then ok
+				if (r.length == 1 && r >= a && r <= z) return r;
+
+				// try to map integers 10-35 to [a-z]
+				i = parseInt(r);
+				if (isNaN(i) || i < 10 || i > 35) throw 'Numeric choice/matching response elements must use a value from 0 to 35 in SCORM 1.2';
+
+				return String.fromCharCode(base + i - 10); // 10 maps to 'a', 11 maps to 'b', ..., 35 maps to 'z'
```

Comment:
You can use `Number(i).toString(36)` instead to convert. It's base 36 instead of base 10.

Reply:
Resolved